heka/usr/share/heka/lua_filters/nginx_redshift.lua

```diff
@@ -69,8 +76,8 @@ function process_message()
         return 0
     end
 
-    local version = string.sub(fields[2], 1, 2)
-    if version ~= "v6" and version ~= "v7" and version ~= "v8" then
+    local version, funnelcake = parse_version(fields[2])
+    if not version or version < 6 or version > 8 then
```

Comment:
I'd rather not attempt to parse `version > 8` since we currently should not be sending anything above that at the moment, that and there's an open conversation about `v9` maybe changing some of the field types in which case we'd blow up anyway.

Reply:
Yet are adding `version >= 8` checks later on, so this kind of incompatibility is potentially going to be troublesome. Still, as long as it's deliberate it seems fine to me.